File: KoRadio/KoRadio.Services/Database/Order.cs
using System;
using System.Collections.Generic;

namespace KoRadio.Services.Database;

public partial class Order
{
    public int OrderId { get; set; }

    public int OrderNumber { get; set; }

    public int UserId { get; set; }

    public bool IsCancelled { get; set; }

    public bool IsShipped { get; set; }

    public DateTime CreatedAt { get; set; }

    public virtual ICollection<OrderItem> OrderItems { get; set; } = new List<OrderItem>();

    public virtual User User { get; set; } = null!;
}

File: KoRadio/UI/ko_radio_desktop/lib/models/order.dart
import 'package:json_annotation/json_annotation.dart';
import 'package:ko_radio_desktop/models/order_items.dart';
import 'package:ko_radio_desktop/models/user.dart';


part 'order.g.dart';

@JsonSerializable()
class Order {
  int orderId;
  int? orderNumber;
  User? user;
  bool? isCancelled;
  bool? isShipped;
  double? price;
  DateTime? createdAt;
  List<OrderItems>? orderItems;

  Order(this.orderId);

  factory Order.fromJson(Map<String, dynamic> json) => _$OrderFromJson(json);
  Map<String, dynamic> toJson() => _$OrderToJson(this);
  
}